### ISSUE TITLE
Internal error: Undefined class constant 

### ISSUE THREAD

**Issue by mkoske** [phpstan/phpstan-doctrine#211]
The error message is like this: `Uncaught Error: Undefined class constant 'MYSQL_ATTR_INIT_COMMAND' in /[...]/Db.php:23`. The code that causes this, is just an array defining driver options for Doctrine.

```php
$opts = [
    "driverOptions" => [PDO::MYSQL_ATTR_INIT_COMMAND => ...]
];
```

**Comment by ondrejmirtes** [comment]
Try disabling the opcache in your php.ini.

If it doesn't help, please create a small reproducing repository that shows the problem, thanks.

**Comment by ondrejmirtes** [comment]
One more idea I have - try enabling the `pdo_mysql` extension in your php.ini.

**Comment by mkoske** [comment]
Well, `pdo_mysql` wasn't event installed (the app itself runs inside docker, but I have phpstan and some other util on local machine). Thanks for the help!

**Comment by ondrejmirtes** [comment]
Soon-to-be-released PHPStan 1.3.0 addresses this, you can now use OPCache again with it! Feel free to test phpstan/phpstan dev-master to verify :)
